_includes/sidebar.html

```diff
@@ -11,9 +11,9 @@
   </section>
 
   <section class="meta">
-    <a href="{{ site.github.owner_url }}" target="_blank"><i class="fa fa-github"></i></a>
-    <a href="https://twitter.com/{{ site.twitter }}" target="_blank"><i class="fa fa-twitter"></i></a>
-    <a href="/atom.xml"><i class="fa fa-rss"></i></a>
+    <a href="{{ site.github.owner_url }}" target="_blank" title="Follow me on GitHub" aria-label="Follow me on GitHub"><i class="fa fa-github"></i></a>
```

Comment:
You don't need both title and aria-label, and aria-label seems to be preferred: https://www.deque.com/blog/text-links-practices-screen-readers/#:~:text=In%20situations%20where%20text%20that,the%20title%20is%20more%20suitable.